qa/tasks/ceph_manager.py

```diff
@@ -1560,6 +1560,9 @@ def ceph(self, cmd, **kwargs):
         stderr = kwargs.pop('stderr', StringIO())
         return self.run_cluster_cmd(args=args, stdout=stdout, stderr=stderr, **kwargs)
 
+    def cepho(self, *args, **kwargs):
+        return self.ceph(*args, **kwargs).stdout.getvalue()
```

Comment:
``raw_cluster_cmd`` calls ``run_cluster_cmd`` which Does The Right Thing for cephadm, rook, and legacy clusters.  I'm not sure what this is needed for?

Reply:
I think I was fixing:

https://github.com/ceph/ceph/pull/43214/files#diff-a31fb9fe7c629a895231a8bd8f0f3f5874dc251130c571fb0ac40a6911c554ceR129

which was not cephadm aware but then got erroneously carried away "fixing" `filesystem.py`. But actually, `filesystem.py` still won't be able to call `ceph ...` commands (I think) because it does not use the `self.ctx.managers[clustername]` manager. It uses its own:

https://github.com/ceph/ceph/blob/7cf10eb17646fc98fafc05e2f6e220c4e43f3854/qa/tasks/cephfs/filesystem.py#L228

So I believe you are correct. Only the above should be changed to use the `self.ctx` manager instance.

Reply:
Fixed! Also dropped a commit to `qa/tasks/cephadm.py` that was not needed for this PR.